src/SFA.DAS.Assessor.Functions/Domain/Learners/EnqueueLearnerInfoCommand.cs

```diff
@@ -29,12 +29,16 @@ public class EnqueueLearnerInfoCommand : IEnqueueLearnerInfoCommand
             _assessorServiceRepository = assessorServiceRepository;
         }
 
-        public async Task Execute()
+        public async Task Execute(string batchMessage)
         {
             try
             {
                 _logger.LogInformation("EnqueueLearnerInfoCommand started");
 
+                var approvalBatchLearnersCommand = JsonConvert.DeserializeObject<ProcessApprovalBatchLearnersCommand>(batchMessage);
```

Comment:
This deserialization can be handled in the function trigger by changing the type of the incoming message I believe?

Reply:
changed